zap/src/main/resources/resource/icon/breakTypes/sources.txt

```diff
@@ -0,0 +1,5 @@
+<div>Icons made by <a href="https://www.flaticon.com/authors/smashicons" title="Smashicons">Smashicons</a> from <a href="https://www.flaticon.com/"                 title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"                 title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
```

Comment:
- This should be moved to `zap/src/main/dist/license/`.
 - Better have this as plain text (per file extension).
 - To be easier/clear it should be enumerated which icons/files apply to which licenses (and link to the icon set). Was any icon changed?

Reply:
I may have resized them and I have created a a gray-scale version of each

Reply:
OK, we should note that then.